### PR TITLE
News announcement for website, docs and 2.5.2

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

File: news/_posts/2017-05-24-akka-2.5.2-released.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+short: Akka 2.5.2 released
+---
+
+Dear hakkers,
+
+We are excited to announce not only a new patch release of Akka 2.5 but also brand new [akka.io](http://akka.io/) website and documentation.
+
+## New documentation
+
+One conclusion from the [community survey 2016](http://blog.akka.io/community/2017/01/17/community-survey-2016-summary) was that the getting started experience of Akka must be improved. Therefore we have written completely new [Getting Started](http://doc.akka.io/docs/akka/current/scala/guide/introduction.html) and [Quickstart](http://akka.io/try-akka/) guides.
```
Comment (raboof) [inline news/_posts/2017-05-24-akka-2.5.2-released.md]: Still need new format for blog links

File: news/_posts/2017-05-24-akka-2.5.2-released.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+You find the landing page for the documentation at [http://akka.io/docs/](http://akka.io/docs/) 
+
+The documentation tooling has been changed to [Paradox](https://github.com/lightbend/paradox). The markdown syntax and the zero installation will make it easier for contributors to help out with improving the documentation. Such contributions are very welcome and you find instructions in [CONTRIBUTING.md](https://github.com/akka/akka/blob/master/CONTRIBUTING.md#documentation).
+
+## Akka 2.5.2
+
+If you are using Cluster Sharding with "remembering entities" there is one important change that requires that you make a change to your code before updating to 2.5.2. To solve a [critical issue #22289](https://github.com/akka/akka/issues/22289) that could result in duplicate entities when the extractShardId function is changed (e.g. between deployments) there is now a new message Shard.StartEntity(entityId) that the extractShardId must understand. See example in [Scala](http://doc.akka.io/docs/akka/current/scala/cluster-sharding.html#remembering-entities) or [Java](http://doc.akka.io/docs/akka/current/java/cluster-sharding.html#remembering-entities).
```
Comment (patriknw) [inline news/_posts/2017-05-24-akka-2.5.2-released.md]: s/before updating/when updating/

Comment (patriknw) [inline news/_posts/2017-05-24-akka-2.5.2-released.md]: some backticks around extractShardId and Shard.StartEntity(entityId)

File: news/_posts/2017-05-24-akka-2.5.2-released.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+      1      1        1 Alena Varkockova
+      1      1        1 André Laszlo
+      1      1        0 Packt
+      1      0        0 synox
+      1      0        0 Dale Wijnand
+
+Happy hakking!
+
+– The Akka Team
+
```
Comment (johanandren) [inline news/_posts/2017-05-24-akka-2.5.2-released.md]: File should go under blog/_posts with the new site

File: news/_posts/2017-05-24-akka-2.5.2-released.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
+---
+layout: news
+title: New documentation, website and Akka 2.5.2 released
+author: Patrik Nordwall
+short: Akka 2.5.2 released
```
Comment (johanandren) [inline news/_posts/2017-05-24-akka-2.5.2-released.md]: Double check these attributes with the ported 2.5.1 announcement, I think there are changes.

File: blog/_posts/2017-05-24-akka-2.5.2-released.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+The development branch of Akka Typed has been merged back to master and is included in Akka 2.5.2, see the blog post series starting with [Akka Typed: Hello World in the new API](http://akka.io/blog/2017/05/05/typed-intro.html). The APIs may still change but we are confident that most of it is now stable. We will continue the work by integrating with other parts of Akka, such as Streams, Persistence, and Cluster.
+
+A total of 77 issues were closed since 2.5.1. The complete list of closed issues can be found on the [2.5.2](https://github.com/akka/akka/milestone/111?closed=1) and [2.5.99-TYPED-M1](https://github.com/akka/akka/milestone/114?closed=1) milestones on github.
+
+## Credits
+
+For this release we had the help of 27 committers – thank you all very much!
+
+commits  added  removed
```
Comment (patriknw) [inline blog/_posts/2017-05-24-akka-2.5.2-released.md]: missing 3-`

**Pull request merged**